travis-scripts/script.sh

```diff
@@ -4,14 +4,6 @@ set -x
 
 cd $TRAVIS_BUILD_DIR || exit 1
 
-# Unshallow the clone. Fetch the tags from upstream even if we are on a
-# foreign clone. Needed for determine-version.py to work, specifically
-# `git describe --tags HEAD` was failing once the last tagged commit
-# became too old.
-git fetch --unshallow
-git remote add upstream https://github.com/cfengine/core.git  \
-    && git fetch upstream 'refs/tags/*:refs/tags/*'
-
```

Comment:
Trying to sneak things in? :wink:

Reply:
No, it was there because determine-version needed it.

Reply:
It even says so in the comment.

Reply:
OH, wow, that didn't even cross my mind!